### PR TITLE
changing file names

### LINKED ISSUE #663: Refactor file generation scripts, and rename files for 2018 and 2019.
The issue is complete with the following tasks. 
- [x] Refactor`generate_clean_files.py`to use the FileGenerator class. 
- [x] Refactor `generate_error_files.py` to use the FileGenerator class, and provide the capability for quality files to pass syntax and validity edits.

### PR DESCRIPTION
**Pull request by MichaelFurey17**
close #663

### CODE REVIEW

File: 2019/python/generate_2019_clean_files.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#This script is used to create test files taht pass all syntax and validity edits that do not require panel data.
+#Configuration of the generated data is done in config.yaml.
+#This script will write files to the relative path "../edits_files/"
+#This script must be run before generate_error_files.py as that script reiles on the presence of a clean file to modify.
```
Comment (Kibrael) [inline 2019/python/generate_2019_clean_files.py]: *relies

File: 2018/python/generate_2018_error_files.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#This script will generate a file that will fail a specific edit (though others may also fail)
+#This script relies on the presence of a clean data file
```
Comment (Kibrael) [inline 2018/python/generate_2018_error_files.py]: Can you list which edit types? Is it all S/V/Q?

File: 2019/python/generate_2019_clean_files.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#This script is used to create test files taht pass all syntax and validity edits that do not require panel data.
```
Comment (Kibrael) [inline 2019/python/generate_2019_clean_files.py]: *that

File: 2018/python/generate_2018_error_files.py (hunk 2 above)
Comment (MichaelFurey17) [reply 2018/python/generate_2018_error_files.py]: Yes, I just did in the last commit.

**Pull request merged**